File: 1276-number-of-burgers-with-no-waste-of-ingredients/1276-number-of-burgers-with-no-waste-of-ingredients.py
class Solution:
    def numOfBurgers(self, tomatoSlices: int, cheeseSlices: int) -> List[int]:
        if (tomatoSlices & 1):
            return []
        
        jumbo = ( tomatoSlices - 2 * (cheeseSlices) ) // 2
        small = cheeseSlices - jumbo
        
        return [jumbo, small] if jumbo >= 0 and small >= 0 else []

File: 1276-number-of-burgers-with-no-waste-of-ingredients/1276-number-of-burgers-with-no-waste-of-ingredients.java
class Solution {
    public List<Integer> numOfBurgers(int tomatoSlices, int cheeseSlices) {
        if((tomatoSlices&1) == 1)  return Collections.emptyList();
        
        int jumbo = ( tomatoSlices - 2 * (cheeseSlices) ) / 2;
        int small = cheeseSlices - jumbo;
        
        return (jumbo >= 0 && small >= 0) ? Arrays.asList(jumbo, small) : Collections.emptyList();
    }
}